.circleci/config.yml

```diff
@@ -190,6 +190,13 @@ workflows:
             branches:
               only: main
 
+      # Notify internal CI about main and release branch updates
```

Comment:
@evil159 @OdNairy would like your input whether `steve` is right workflow to run this `trigger` job

Reply:
Yep, that right.
`Steve` workflow is something we run on each PR/merge.